youtube_dl/extractor/facebook.py

```diff
@@ -492,7 +492,10 @@ def parse_attachment(attachment, key='media'):
                         lambda x: x['attachments']
                     ], list) or []
                     for attachment in attachments:
-                        attachment = try_get(attachment, lambda x: x['style_type_renderer']['attachment'], dict)
+                        attachment = try_get(attachment, [
```

Comment:
Would it be better style for the getters to be a tuple, immutable, rather than a list?

Reply:
This style matches the code around. There also seems to be many use of list in other places like `self._search_regex()` and coding convention examples.
If `for x in ...`, then tuple would be preferred.